ytmusicapi/mixins/browsing.py

```diff
@@ -876,7 +1068,7 @@ def get_basejs_url(self):
         if match is None:
             raise YTMusicError("Could not identify the URL for base.js player.")
 
-        return YTM_DOMAIN + match.group(1)
+        return cast(str, YTM_DOMAIN + match.group(1))
```

Comment:
please don't change unrelated functions

Reply:
ok, could you either revert the change or add the return type then? I'd prefer the latter